tools/python/translate.py

```diff
@@ -149,8 +153,13 @@ def usage():
 
   text_to_translate = ' '.join(sys.argv[1:])
 
-  translations = deepl_translate(text_to_translate)
-  google_translations = google_translate(text_to_translate)
+  source_language = 'en'
+  if len(text_to_translate) > 3 and text_to_translate[2] == ':':
+    source_language = text_to_translate[0:2]
+    text_to_translate = text_to_translate[3:].lstrip()
+
+  translations = deepl_translate(text_to_translate, source_language)
+  google_translations = google_translate(text_to_translate, source_language)
   translations.update(google_translations)
```

Comment:
just here, you are using google's one.

Reply:
DeepL does not support all languages that are already present in OM. Google Translate is used as a fallback for those unsupported languages.